src/@orb.yml

```diff
@@ -153,22 +170,84 @@ commands:
         type: boolean
     steps:
       - run:
-          name: Extract OWASP Dependency Check reports
+          name: Extract OWASP Dependency-Check reports
+          # Run even if a previous step returned non-zero exit code.
+          # The command-line tool returns non-zero exit codes if it finds
+          # vulernabilities above a certian CVSS score is detected.
+          when: always
           command: |
             mkdir -p Reports/OWASP
             cp << parameters.reports_path >>/dependency-check*.* Reports/OWASP
       - store_artifacts:
-          name: Store OWASP Dependency Check reports as artifacts (in 'Reports/OWASP' directory)
+          name: Store OWASP Dependency-Check reports as artifacts (in 'Reports/OWASP' directory)
           path: Reports/OWASP
           destination: Reports/OWASP
       - when:
           condition: <<parameters.persist_to_workspace>>
           steps:
             - persist_to_workspace:
-                name: Persist OWASP Dependency Check reports to workspace (in 'Reports/OWASP' directory)
+                name: Persist OWASP Dependency-Check reports to workspace (in 'Reports/OWASP' directory)
                 root: .
                 paths:
                   - Reports/OWASP
+  with_commandline:
+    description: |
+      Run a set of steps with the OWASP Dependency-Check command-line tool. Run from ~/.owasp/dependency-check/bin/dependency-check.sh.
+    parameters:
+      steps:
+        type: steps
+      version_url:
+        description: Location of latest Dependency-Check command-line tool version (i.e. 5.2.1)
+        type: string
+        default: "https://jeremylong.github.io/DependencyCheck/current.txt"
+      executable_url:
```

Comment:
As an alternative you could expect the executing image to contain dependenct_check.sh. Perhaps even have a separate command for installing owasp.

Reply:
I'll save that for a later version + might be sensible to also look at customizing the command-line arguments also for maven and gradle builds.